src/features/defi/providers/idle/components/IdleManager/Claim/components/Status.tsx

```diff
@@ -0,0 +1,205 @@
+import { CheckIcon, CloseIcon, ExternalLinkIcon } from '@chakra-ui/icons'
+import { Box, Button, Link, Stack } from '@chakra-ui/react'
+import { ASSET_REFERENCE, toAssetId } from '@shapeshiftoss/caip'
+import { Summary } from 'features/defi/components/Summary'
+import { TxStatus } from 'features/defi/components/TxStatus/TxStatus'
+import { DefiParams, DefiQueryParams } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import { useContext, useEffect, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { StatusTextEnum } from 'components/RouteSteps/RouteSteps'
+import { Row } from 'components/Row/Row'
+import { Text } from 'components/Text'
+import { useBrowserRouter } from 'hooks/useBrowserRouter/useBrowserRouter'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import {
+  selectAssetById,
+  selectFirstAccountSpecifierByChainId,
+  selectMarketDataById,
+  selectTxById,
+} from 'state/slices/selectors'
+import { serializeTxIndex } from 'state/slices/txHistorySlice/utils'
+import { useAppSelector } from 'state/store'
+
+import { IdleClaimActionType } from '../ClaimCommon'
+import { ClaimContext } from '../ClaimContext'
+import { ClaimableAsset } from './ClaimableAsset'
+
+export const Status = () => {
+  const translate = useTranslate()
+  const { state, dispatch } = useContext(ClaimContext)
+  const { query, history: browserHistory } = useBrowserRouter<DefiQueryParams, DefiParams>()
+  const { chainId, contractAddress: vaultAddress, assetReference } = query
+
+  const assetNamespace = 'erc20'
+  // Asset info
+  const underlyingAssetId = toAssetId({
+    chainId,
+    assetNamespace,
+    assetReference,
+  })
+  const underlyingAsset = useAppSelector(state => selectAssetById(state, underlyingAssetId))
+  const assetId = toAssetId({
+    chainId,
+    assetNamespace,
+    assetReference: vaultAddress,
+  })
+  const asset = useAppSelector(state => selectAssetById(state, assetId))
+  const feeAssetId = toAssetId({
+    chainId,
+    assetNamespace: 'slip44',
+    assetReference: ASSET_REFERENCE.Ethereum,
+  })
+  const feeAsset = useAppSelector(state => selectAssetById(state, feeAssetId))
+  const feeMarketData = useAppSelector(state => selectMarketDataById(state, feeAssetId))
+
+  const accountSpecifier = useAppSelector(state =>
+    selectFirstAccountSpecifierByChainId(state, chainId),
+  )
+
+  const serializedTxIndex = useMemo(() => {
+    if (!(state?.txid && state?.userAddress)) return ''
+    return serializeTxIndex(accountSpecifier, state.txid, state.userAddress)
+  }, [state?.txid, state?.userAddress, accountSpecifier])
+  const confirmedTransaction = useAppSelector(gs => selectTxById(gs, serializedTxIndex))
```

Comment:
What's `gs` ?

Reply:
Taken from the yearn implementation and also here in the Fox/ETH LP opportunity.
You don't need to update this here, but please keep these comments open, I'll resolve them after creating an issue with all the additional improvements we need for the DeFi UI abstraction.

Reply:
Copied from /src/features/defi/providers/yearn/components/YearnManager/Deposit/components/Status.tsx:

Reply:
https://github.com/shapeshift/web/issues/2434